testutils/src/main/java/com/hyperwallet/android/ui/testutils/rule/HyperwalletTestRule.java

```diff
@@ -0,0 +1,22 @@
+package com.hyperwallet.android.ui.testutils.rule;
+
+import com.hyperwallet.android.Hyperwallet;
+import com.hyperwallet.android.ui.testutils.TestAuthenticationProvider;
+
+import org.junit.rules.TestWatcher;
+import org.junit.runner.Description;
+
+public class HyperwalletTestRule extends TestWatcher {
```

Comment:
We need to keep names consistent. I think the word "Test" shouldn't be part of the name

Reply:
Renamed to HyperwalletSdkRule